operator/check_operator.go

```diff
@@ -35,29 +36,9 @@ func NewCheck(image, indeximage string, kubeconfig []byte, opts ...Option) *oper
 
 // Run executes the check and returns the results.
 func (c operatorCheck) Run(ctx context.Context) (certification.Results, error) {
-	switch {
-	case c.image == "":
-		return certification.Results{}, preflighterr.ErrImageEmpty
-	case c.kubeconfig == nil:
-		return certification.Results{}, preflighterr.ErrKubeconfigEmpty
-	case c.indeximage == "":
-		return certification.Results{}, preflighterr.ErrIndexImageEmpty
-	}
-
-	pol := policy.PolicyOperator
-
-	checks, err := engine.InitializeOperatorChecks(ctx, pol, engine.OperatorCheckConfig{
-		ScorecardImage:          c.scorecardImage,
-		ScorecardWaitTime:       c.scorecardWaitTime,
-		ScorecardNamespace:      c.scorecardNamespace,
-		ScorecardServiceAccount: c.scorecardServiceAccount,
-		IndexImage:              c.indeximage,
-		DockerConfig:            c.dockerConfigFilePath,
-		Channel:                 c.operatorChannel,
-		Kubeconfig:              c.kubeconfig,
-	})
+	_, checks, err := c.List(ctx)
 	if err != nil {
-		return certification.Results{}, fmt.Errorf("%w: %s", preflighterr.ErrCannotInitializeChecks, err)
+		return certification.Results{}, fmt.Errorf("%w: %s", preflighterr.ErrCannotListChecks, err)
```

Comment:
This will fix the failing tests
```suggestion
		return certification.Results{}, err
```

Reply:
fixed